### DISCUSSION TITLE
Support for zellij in preview-tui script

### DISCUSSION THREAD

**Discussion by ih8d8** [jarun/nnn#1790] [Ideas]
Any plan on adding support for [zellij](https://github.com/zellij-org/zellij) in the preview-tui script?

**Comment by arut-plus** [comment]
It would be nice :100:

**Comment by N-R-K** [comment]
> Any plan on adding support for [zellij](https://github.com/zellij-org/zellij) in the preview-tui script?

No plans. And @luukvbaal (main author of preview-tui script) has not been active in a while. If it's important to you, feel free to submit a PR.

**Comment by jarun** [comment]
It would be a very good value addition. We need someone to contribute. A good way to start would be to follow `NNN_TERMINAL` in the `preview-tui` script.
